### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_ScreenReader: After performing any action on checkbox under "Nullable " header, focus doesn't move to any other element.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6168]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS3.3.2; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp 
**Environment Details:**
Test Pass Name: A11y_AzureDataStudio
MAS: MAS3.2.2; MAS1.3.1
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS :Windows_NT x64 10.0.18362
JAWS Version: 2019.1906.10 ILM 

**Additional Details:**
Tools Name: JAWS
MAS References: MAS3.2.2,MAS1.3.1

**Steps to Reproduce:**

1. Install the app Azure Data Studio.
2. Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.
3. Select any tree view item from left panel using arrow key and press enter.
4. Navigate till table using basic navigation 
5. Navigate to checkbox using downward arrow and perform action using spacebar.
6. Observe that after performing any action on checkbox under "Nullable " header, screen reader focus doesn't move to any other element.

**Actual:**
While navigating table using downword arrow, after performing any action on checkbox under "Nullable " header, screen reader  focus doesn't move to any other element and gets trapped on that checkbox.

**Expected:**
After performing any action on checkbox under "Nullable " header, Screen reader focus should move to next column that is "Collations".

**User Impact:**
Screen reader only users find it difficult to navigate the table because if they are using basic navigation that is up/down arrow, then after performing any action they unable to move on next column .

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference :
MAS3.2.2:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={24cbfff7-374e-42e6-b9ab-d87fcd53b3e8}
**MAS1.3.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference:
[6168_[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3328423/6168_.Accessibility.A11y_AzureDataStudio_MapDataSrcObjects_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by udeeshagautam** [comment]
Seems similar to this issue #6800
Should be fixed in this PR  https://github.com/microsoft/azuredatastudio/pull/6882

**Comment by Charles-Gagnon** [comment]
@AccessibilityTestingTeam-TCS I'm closing this since it seems to be working with tab navigation. The issue mentions using down arrow to navigate though - is that required in addition to tab navigation?
